fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/synchronizedsession/SynchronizedSessionRunner.java

```diff
@@ -163,6 +167,28 @@ public void endSession() {
         underlying.asyncToSync(FDBStoreTimer.Waits.WAIT_END_SYNC_SESSION, endSessionAsync());
     }
 
+    /**
+     * Releases the lock to end any synchronized session on it, no matter the current session holds the lock or not.
```

Comment:
```suggestion
     * Releases the lock to end any synchronized session on the same subspace, no matter whether the current session holds the lock or not.
```

Reply:
I think? (I think I'm a little confused what the antecedent of "it" is meant to be.)

Reply:
A minor point (one you are absolutely welcome to ignore), but most of the comments use `lock` when I think the more correct wording would be `lease`.

Reply:
I differentiated `lock` and `lease` in this way: A session is allowed to work even if its **lease** is outdated (= not keeping the lease), as long as no other session takes its **lock**. In this sense, I should use "lock" here. But it's likely I may have misused them somewhere.

Reply:
Ok, fair point.  Thanks for clarifying!